wherescape/connectors/hubspot/process_data.py

```diff
@@ -0,0 +1,155 @@
+import logging
+from decimal import Decimal
+from datetime import datetime
+from .hubspot_wrapper import Hubspot
+
+# from helper_functions import compare_names
+
+"""
+This module processes the collected data so it can be send to the Hubspot Module
+"""
+
+
+def hubspot_process_results(
+    access_token: str, results: list, column_names: list, table_name: str
+):
+    """
+    function that handles the processing of the results for it to be send to Hubspot
+    """
+    hubspot_instance = Hubspot(access_token)
+    properties = []
+
+    object_name = get_object_name(table_name)
+    request_type = get_http_request_type(table_name)
+    property_names = hubspot_instance.get_company_properties()
```

Comment:
line 22 could also be deals or contacts. why only `get_company_properties` here?